pkg/api/helpers.go

```diff
@@ -141,3 +141,42 @@ func HashObject(obj runtime.Object, codec runtime.Codec) (string, error) {
 	}
 	return fmt.Sprintf("%x", md5.Sum(data)), nil
 }
+
+func (l *LoadBalancerStatus) Equal(r *LoadBalancerStatus) bool {
```

Comment:
We sort of have a thing against API objects having methods.  Not sure why at this point, but I don't want to re-adjudicate it right now.  Propose just renaming this and DeepCopy and leaving a TODO.